### PR TITLE
link added on name- cusror pointer

### PR DESCRIPTION
**Pull request by Vamthecoder**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,18 +14,18 @@
    </header>
```
Comment (ankur-1989) [inline index.html:14]: there is no need of `p` tag inside `header` tag.

**Comment by ankur-1989** [comment]
if max-width is 100% then you can remove it. It is unnecessary. 
https://github.com/Vamthecoder/dev-challenges-404/pull/1/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R21

**Comment by ankur-1989** [comment]
Here also, https://github.com/Vamthecoder/dev-challenges-404/pull/1/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R32 - justify-items is not doing anything. as you have used text-align left there. so you can remove justify-items: left and check if this breaks anything.

**Comment by ankur-1989** [comment]
https://github.com/Vamthecoder/dev-challenges-404/pull/1/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R83
max-height: 50%  - here  50% of what ?  even if you remove this, it will still work

**Comment by ankur-1989** [comment]
1. CSS code for main inside media query 850px and 450px is same and duplicate so you can write in 850 only and it will work for 450px as well. 
2. for Button- max-width and min-width: max-content and min-content, what is the purpose of these two. I think just saying width: fit-content is enough. as you are giving padding to give some room inside the button.

File: index.html (hunk 1 above)
Comment (Vamthecoder) [reply index.html:14]: Actually P has margin-block of 1 rem by default. header has no margin. i have to give it to header. either way

**Comment by Vamthecoder** [comment]
> https://github.com/Vamthecoder/dev-challenges-404/pull/1/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1R83 max-height: 50% - here 50% of what ? even if you remove this, it will still work

Well if you go responsive. it will go under the image. it will automatically take appropriate height and gap. so leaving it there. 50% of the parent element. as i have used grid. that will work that way

**Pull request closed without merging**